packages/compass-schema/src/components/compass-schema.tsx

```diff
@@ -373,6 +374,7 @@ const Schema: React.FunctionComponent<{
           />
         }
       >
+        <HackoladePromoBanner></HackoladePromoBanner>
```

Comment:
Should this ever go away or be only shown on the initial `ANALYSIS_STATE_INITIAL` screen? It looks like it's always going to be taking up some screen space.

Reply:
It will, this is requirement [per scope](https://docs.google.com/document/d/1HPJTiKyaPnwAH3SIAkdGPBqxZ92ALCtX2Cm24kzCLYQ/edit#heading=h.iveagqw4hj80) and [design](https://www.figma.com/file/i96JS6BlH4HesWQ11dLDIj/EXPO-2311-%3A-Compass-CTA-for-Hackolade?type=design&node-id=356%3A4075&t=d4Eo2EDSikP3ZJyB-1) (sorry, should've left a link to this initially!). It will not be in the way when you scroll though, so shouldn't be that bad